reference/pipeline/expressions/index.md

```diff
@@ -186,12 +186,36 @@ information from Jenkins properties files or other similar endpoints.
 
 ### #stage(String)
 
-A shortcut to get the stage by name. For example, `${#stage("Bake")}` allows
+A shortcut to get the stage by name. For example, `${#stage('Bake')}` allows
```

Comment:
@marchello2000 Any reason you changed `'` to `"` in this first occurrance, but not in subsequent occurrences in this paragraph?

Reply:
my preference is `'` but I will just stick to convention and use `"` here